attest/verifier/build.rs

```diff
@@ -50,10 +48,8 @@ lazy_static::lazy_static! {
             },
             Err(_) => {
                 cargo_emit::warning!(
-                    "Using thread_rng() to generate mock attestation report signatories for simulation-mode enclaves"
+                    "Using default seed to generate mock attestation report signatories for simulation-mode enclaves"
                 );
-                let mut csprng = rand::thread_rng();
-                csprng.fill_bytes(&mut seed[..]);
```

Comment:
This was discussed offline, we don't think there's any benefit to using a random seed for sim builds since they aren't secure. The random seed per generation causes issues at times trying to attest, due to the timing of enclaves and untrusted executing this build script concurrently